cmd/tempo-query/tempo/config.go

```diff
@@ -1,15 +1,22 @@
 package tempo
 
 import (
+	"github.com/jaegertracing/jaeger/plugin/storage/grpc/shared"
 	"github.com/spf13/viper"
 )
 
 // Config holds the configuration for redbull.
 type Config struct {
-	Backend string `yaml:"backend"`
+	Backend      string `yaml:"backend"`
+	TenantHeader string `yaml:"tenantHeader"`
```

Comment:
In tempo we use snake_case for YAML names. Also, would it be better if it was named `tenant_header_key`?

Reply:
fixed